UniAtHome/UniAtHome.BLL/UniAtHome.BLL.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I have changed TargetFramework to netcoreapp3.1 because Microsoft.Identity.EntityFrameworkCore package does not support netstandard2.0? Is it problem for you to use newer version of framework?

Reply:
Hell yeah, I can't install it due to my system problems